src/draw.cpp

```diff
@@ -69,15 +69,13 @@ void draw::fill(const SDL_Rect& area)
 
 void draw::set_color(uint8_t r, uint8_t g, uint8_t b, uint8_t a)
 {
-	DBG_D << "set color "
-	      << " [" << r << ',' << g << ',' << b << ',' << a << ']' << endl;
+	DBG_D << "set color " << color_t{r,g,b,a} << endl;
 	SDL_SetRenderDrawColor(renderer(), r, g, b, a);
 }
 
 void draw::set_color(uint8_t r, uint8_t g, uint8_t b)
 {
-	DBG_D << "set color "
-	      << " [" << r << ',' << g << ',' << b << ']' << endl;
+	DBG_D << "set color " << color_t{r,g,b,SDL_ALPHA_OPAQUE} << endl;
```

Comment:
Doesn't it also work if you omit the `SDL_ALPHA_OPAQUE`?

Reply:
good point, yes it does